cfpb/sbl-frontend
# [e2e] [Complete user profile - unassociated] Ensure first name / last name have char limits

Characters limits and unicode character input tolerance need to be validated in an e2e test.

## Changes

- Added unicodeConstants.ts file so that special Unicode Constants have somewhere to go.
- Updated NonAssociatedUserProfile.spec.ts with a new test for character limit validation

## How to test this PR

1. Pull the branch
2. Run the e2e tests and verify that the 'Complete User Profile with Bad Unicode -- No Associations -- process' test passes
